Gordon360/Enums/AuthGroup.cs

```diff
@@ -13,7 +13,14 @@ public enum AuthGroup
     RecIMSuperAdmin,
     SiteAdmin,
     Staff,
-    Student
+    Student,
+    RA,
+    RD,
+    HallInfoViewer,
+    HousingDeveloper, //Remove before deployment
```

Comment:
@RossClark01 are we removing this before deployment?

Reply:
We will likely still want this permission group for testing purposes at first despite the comment. This will allow us as developers to keep an eye on all the views in production